tests/test_state.py

```diff
@@ -92,3 +97,66 @@ def test_state_serialize(
 
     # both states should have equivalent state, model parameters, loss, and outputs
     assert_state_equivalent(state1, state2)
+
+
+my_list = [3, 4, 5, 6, 7, 8, 9, 10]
```

Comment:
Since the state helpers use the `batch_get` / `batch_set` helpers, no need for an extensive matrix here. It's best that each test does only one thing -- here that would be to ensure that these methods exist on the state and call the helpers. Perhaps here we could monkeypatch `batch_get` / `batch_set` to use dummy values, and then assert that `state.batch_get_item(...)` returns the dummy value, and after `state.batch_set_item(...)`, `state.batch` is then the dummy batch?

Reply:
Good call! I added a pair of mock.patch tests here using `return_value` and `assert_called_once` here: [b74ebd1](https://github.com/mosaicml/composer/pull/978/commits/b74ebd19e5d87682b6bdb41d31e91ff97ea15716)